sound/effects/radiofreq/Sound License.txt

```diff
@@ -0,0 +1,29 @@
+sci.ogg
+[80s laser gun] obtained from https://youtu.be/IQ-HeEhE5nY
+
+com.ogg  (and centcomm)
+[chime] sampled from sound/ai/intern's chime
+
+med.ogg
+[sos beeping] Obtained from pixaby, royalty free
+
+eng.ogg
+[sparks and generic radio crackle] Obtained from pixaby, royalty free
+
+sec.ogg
+[combine radio sfx] Obtained from Half Life 2's combine radio sounds.
```

Comment:
I like this sound the most and wouldn't be against security specifically having it, however I don't think we can use sounds taken from HL2 since they are licensed.

Reply:
Well apparently they are licensed for non-commercial use for anyone that owns the game license.